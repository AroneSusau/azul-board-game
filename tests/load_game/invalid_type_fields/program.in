2
tests/load_game/invalid_type_fields/testing.azul
