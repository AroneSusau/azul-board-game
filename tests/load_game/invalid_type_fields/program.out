[H[2J
[34m       db[35m   8888P[31m 8    8[33m 8    
[34m     dPYb[35m    dP  [31m8    8[33m 8     
[34m   dPwwYb[35m  dP   [31m8b..d8[33m 8      
[34m dP    Yb[35m d8888 [31m`Y88P'[33m 8888   
[0m
Enter a number
[37m1.[2m Start Game[0m
[37m2.[2m Load Game[0m
[37m3.[2m Credits[0m
[37m4.[2m Exit[0m
[0m[32m> [0m[H[2JEnter the filename from which load a game
[32m> [0m
[31mError: Unexpected type 'cat' detected.[0m

[2mPress enter to continue..[0m[H[2J[H[2J
[32m __                      _        _                
[32m //_ _  _  /_ /_/_      /_`_  _  /_//_     ._  _  /
[32m// //_|/ //\  / /_//_/ /  /_//  /  //_|/_/// //_/. 
[32m                                       _/     _/   
[0m
[0m